Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/countdown/Countdown.tsx

```diff
@@ -24,20 +25,63 @@ interface ICountdownProps {
     handleOnCountdownChange?: (seconds: number) => void;
 }
 
+const remainingTimeClassName = 'remainingTime';
+
+const renderTimeOut = () => (
+    <p className={remainingTimeClassName}>
+        Remaining time:
+        {' '}
+        <Text type={TextType.Bold}>
+            {' '}
+            00, 00, 00
+        </Text>
+    </p>
+);
+
 const defaultRender = (remainingTime: ICountdownRemainingType) => {
```

Comment:
You can make this function also to be a useCallback.

Reply:
After discussion we agreed this doesn't need to be a useCallback function.